met-web/src/components/common/Table/index.tsx

```diff
@@ -179,18 +119,17 @@ function EnhancedTable({ filter = { key: '', value: '' } }) {
             return;
         }
 
-        const rowsFilteredResults = rows.filter((row: Engagement) => {
+        const rowsFilteredResults = rows.filter((row: T) => {
             if (!hasKey(row, filter.key)) {
                 return false;
             }
             // filter by rows who have the specified field in filter.key matching the search filter value
-            return String(row[filter.key]).toLowerCase().match(`${filter.value.toLowerCase()}.*`);
```

Comment:
I think this reintroduces the "case sensitive" filter bug

Reply:
Will it? Noted, this was pulled from main. Would you like to add a bug ticket for it?